src/omnibar/omnibar.ts

```diff
@@ -63,6 +63,11 @@ function addIframeEl(): void {
 }
 
 function isModernTheme(): boolean {
+  const qs = parseQuerystring();
+  if (qs.forceModern === '1') {
```

Comment:
I like to keep parts of a URL lower-case when possible, including query string parameter names. As this gets more traction we'll probably start telling people "just put this thing in the query string" and having mixed casing is an opportunity for people to input it incorrectly and then go asking why it didn't work.

Also I'd prefer the parameter name indicate that its scope is limited to omnibar (`forceModern` sounds like it would apply to the whole page and not just omnibar). Since we have `leftnav`, how about `modernnav` to match?